std_msgs/Header header
int16 discrepointpersecond
int16 inicounter
int16 arraylength
float32 error
float32[] thrustarray
float32[] thetaarray
float32[] stateXarray
float32[] stateZarray
float32[] stateVXarray
float32[] stateVZarray